posthog/migrations/0218_team_aggregate_users_by_distinct_id.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.5 on 2022-03-08 23:01
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Adding this onto the team model is not IMO the right way - this is a temporary hack to please one enterprise client at best, _not_ something we want to carry forward.

Instance settings will work well for this - use constance to store a list of team_ids for this hack.